src/SprykerEco/Zed/AmazonQuicksight/Business/ApiClient/AssetBundleAmazonQuicksightApiClient.php

```diff
@@ -262,4 +291,17 @@ protected function createQuicksightDescribeAssetBundleImportJobRequestTransfer(
             ->setAssetBundleImportJobId($assetBundleImportJobId)
             ->setAwsAccountId($this->amazonQuicksightConfig->getAwsAccountId());
     }
+
+    /**
+     * @param string $idDataSet
+     *
+     * @return \Generated\Shared\Transfer\QuicksightDeleteDataSetRequestTransfer
+     */
+    protected function createQuicksightDeleteDataSetRequestTransfer(
```

Comment:
There is not much profit from this method, you can keep object creation in the main flow (same time it's being used only once)